docs/config/_default/menus.toml

```diff
@@ -21,7 +21,7 @@
   weight = 10
 
 [[docs]]
-  name = "Installing a Controller"
+  name = "Getting Started"
   identifier = "installing"
```

Comment:
I'm cool with "Getting Started". Let's change the identifier to `"getting-started"` or else it'll confuse the crap out of everyone who wants to add a new page.